bundles/org.openhab.binding.feed/src/main/java/org/openhab/binding/feed/internal/handler/FeedHandler.java

```diff
@@ -100,6 +104,17 @@ private void checkConfiguration() {
                     DEFAULT_REFRESH_TIME, e.getMessage());
             refreshTime = DEFAULT_REFRESH_TIME;
         }
+
+        try {
+            numberOfEntries = (BigDecimal) configuration.get(NUMBER_OF_ENTRIES);
+            if (numberOfEntries.intValue() <= 0) {
+                throw new IllegalArgumentException("Refresh time must be positive number!");
+            }
+        } catch (Exception e) {
```

Comment:
If you want to keep it, please check if you get a value, if it is `instanceof BigDecimal` anduse the range check afterwards. Catching `Exception` is a bad idea in almost all situations as it hides programming errors (like in this case, calling `.intValue()` could result in an null-pointer-exception because `.get()` is allowed to return `null`).